src/librustdoc/formats/renderer.rs

```diff
@@ -38,10 +38,14 @@ crate trait FormatRenderer<'tcx>: Clone {
     fn mod_item_out(&mut self, item_name: &str) -> Result<(), Error>;
 
     /// Post processing hook for cleanup and dumping output to files.
-    fn after_krate(&mut self, krate: &clean::Crate, cache: &Cache) -> Result<(), Error>;
-
-    /// Called after everything else to write out errors.
-    fn after_run(&mut self, diag: &rustc_errors::Handler) -> Result<(), Error>;
+    ///
+    /// A handler is available if the renderer wants to report errors.
```

Comment:
I think this should mention _when_ will the handler be called to report errors. In this case it should be after post processing.

Reply:
That is not guarenteed by this trait, that's an implementation detail of `html::render::Context`. So I don't think it should be documented.